components/Ink/src/MDCInkView.m

```diff
@@ -81,8 +81,10 @@ - (void)commonMDCInkViewInit {
 - (void)layoutSubviews {
   [super layoutSubviews];
 
-  // If the superview has a shadowPath make sure ink does not spread outside of the shadowPath.
-  if (self.superview.layer.shadowPath) {
+  if (self.inkStyle == MDCInkStyleUnbounded) {
+    self.layer.mask = nil;
+  } else if (self.superview.layer.shadowPath) {
```

Comment:
do we need to update the mask if the client decides to change the inkStyle?

Reply:
I don't thiiiink so. This change is in layoutSubviews, which gets called as soon as the ripple begins, so I don't think we need to worry about that.